src/terrain/routes/schemas/qms.clj

```diff
@@ -9,12 +9,12 @@
 (def GetPlanDescription "Returns details about a single plan in QMS. Plan is referenced by its UUID")
 (def GetResourceTypesSummary "List Resource Types")
 (def GetResourceTypesDescription "Returns a list of all resource types registered in QMS")
-(def GetUserPlanSummary "Returns details about the user's current plan")
-(def GetUserPlanDescription "Returns details about the user's current plan, including quota and usage information")
-(def UpdateUserPlanQuotaSummary "Updates a quota in the user's current subscription")
-(def UpdateUserPlanQuotaDescription "Updates the user's quota for the specified resource type")
-(def UpdateUserPlanSummary "Changes the user's plan to another one in QMS")
-(def UpdateUserPlanDescription "Changes the user's plan to another one in QMS. New plan is referenced by name")
+(def GetSubscriptionSummary "Returns details about the user's current plan")
+(def GetSubscriptionDescription "Returns details about the user's current plan, including quota and usage information")
+(def UpdateSubscriptionQuotaSummary "Updates a quota in the user's current subscription")
+(def UpdateSubscriptionQuotaDescription "Updates the user's quota for the specified resource type")
+(def UpdateSubscriptionSummary "Changes the user's plan to another one in QMS")
+(def UpdateSubscriptionDescription "Changes the user's plan to another one in QMS. New plan is referenced by name")
```

Comment:
These descriptions could be reworded a bit to reflect the change as well.

Reply:
This should be fixed now.